src/failures/src/failures-orchestrator.ts

```diff
@@ -9,9 +11,10 @@ export interface Failure {
 /**
  * Orchestrates the activation and deactivation of failures.
  *
- * Only a single instance of the orchestrator should exist within the whole application.
+ * Only a single instance of the orchestrator should exist within the whole application. Its data can be accessed using
+ * a {@link RemoteFailuresProvider}
  */
-export class FailuresOrchestrator {
+export class FailuresOrchestrator implements FailuresProvider {
```

Comment:
**thought**: I have the feeling we're missing some abstractions in these changes. Now this type needs to know about `Coherent`'s `on`, `RegisterViewListener`, and publishing. Maybe there's some additional work here?